### PR TITLE
Fixed issues with the environment variables

### LINKED ISSUE #11: Use ENV files to set token

### PR DESCRIPTION
**Pull request by OG-Jons**
With these fixes, the NODE_ENV var gets set depending on which script you run:
dev => development
start => production
The app utilizes dotenv if it's in development, which allows you to use .env files. I added an example file to show how to use this.

### CODE REVIEW

File: src/Main.ts [hunk 1]
```diff
@@ -23,7 +24,7 @@ async function start() {
     // Glob string for loading classes
     classes: [`./commands/*.{js,ts}`],
     // If you only want to use guild commands, uncomment this line
-    // botGuilds: [(client) => client.guilds.cache.map((guild) => guild.id)],
+    botGuilds: process.env.NODE_ENV ? ["930471643594240023"] : undefined,
```
Comment (SemiShogun) [inline src/Main.ts]: What GuildID is this?

Comment (OG-Jons) [reply src/Main.ts]: oops, overlooked...
it's the GuildId of the funni bot server, where I tested the bot

Comment (SemiShogun) [reply src/Main.ts]: since it's an array we could probably also add in our server guild ID
that way we can test it on both servers, it's a winwin

Comment (SemiShogun) [reply src/Main.ts]: we kinda need this botGuilds attribute to make the bot automatically load in all slash commands once it runs

Comment (OG-Jons) [reply src/Main.ts]: Aight, I'll add that in instead...
just noticed an error in the README file aswell, so I'll fix that

Comment (OG-Jons) [reply src/Main.ts]: I did some testing, with this option enabled like this, it automatically adds the commands to the servers it's connected to...
The command is basically doing the same thing as we are doing when hardcoding... so it's your call...

Comment (SemiShogun) [reply src/Main.ts]: ok then let's leave it the way it was
I'll merge once you have committed the readme file as well

Comment (OG-Jons) [reply src/Main.ts]: 👍👍👍

**Pull request merged**